packages/react-redux/src/slices/cartoSlice.js

```diff
@@ -148,10 +149,15 @@ export const createCartoSlice = (initialState) => {
         }
       },
       removeFilter: (state, action) => {
-        const { id, column } = action.payload;
+        const { id, column, type } = action.payload;
```

Comment:
I'm not sure about using the `type` for that… What about using the `owner` property instead? I think it was made just for this kind of use case: identifying which filter belongs to which widget.

Reply:
yes, I know that would be better to use `owner` instead of `type`. The current structure of filters is using `type` as key, `owner` is a property in the filter, we can't apply more than one filter per type, widgets only applies one type of filter.

IMO, we can delete using `owner` now but has you say in the previous comment we should consider to refactor the filters structure. Maybe using `owner` instead of `type` as key¿?

Reply:
@bbecquet Finally I've decided to use `owner` instead.